templates/common/gcp/files/usr-local-bin-update-dns-server.yaml

```diff
@@ -0,0 +1,22 @@
+mode: 0755
+path: "/usr/local/bin/update-dns-server"
+contents:
+  inline: |
+    #!/bin/bash
+    # This is test code
+    
+    echo "Executing update-dns-server"
+
+    if ![-f /etc/NetworkManager/conf.d]; then
+      mkdir /etc/NetworkManager/conf.d
```

Comment:
Instead of `if [...] then mkdir` you can just do `mkdir -p [...]`. The `-p` flag will make `mkdir` *not* return an error if the directory already exists